fixes.inc

```diff
@@ -6406,6 +6498,31 @@ native BAD_TogglePlayerControllable(playerid, toggle) = TogglePlayerControllable
 #endif
 
 /*
+ * FIXES_StopPlayerEnterVehicle(playerid, bool:kick)
+ *
+ * Block a player from entering a vehicle after the animation for a locked driver door
+ *
+ * FIXES:
+ *     OnPlayerEnterVehicle_3
+ */
+
+#if FIX_OnPlayerEnterVehicle_3
+	static stock FIXES_StopPlayerEnterVehicle(playerid)
```

Comment:
Damn and I saw that when I was going through the file and forgot it then!
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/4cdd88d0132a87f3cc6f99fbae52f5df73ae5b88